music21/key.py

```diff
@@ -1078,7 +1078,7 @@ def deriveByDegree(self, degree, pitchRef):
         To use the harmonic form, change `.abstract` on the key to
         another abstract scale:
 
-        >>> minorKey.abstract = scale.AbstractHarmonicMinorScale()
+        >>> minorKey.abstract = key.Key(mode='harmonic minor').abstract
```

Comment:
This strikes me as a strange way of deriving a scale, i.e., creating a Key object, throwing it away, and then getting the .abstract. Either way we're exposing the user to the concept of abstract scales, so I'm not sure this is an improvement.